cookiecutter/generate.py

```diff
@@ -10,6 +10,7 @@
 from __future__ import unicode_literals
 import logging
 import os
+import os.path as osp
```

Comment:
Gotcha. I'm lazy too. :smile: 

That said, cookiecutter's pattern is to write out the entire statement and not use aliases. Especially for the `os` library, which is used in nearly every module of the project.